packages/components/src/range-control/input-range.js

```diff
@@ -58,8 +76,8 @@ function InputRange(
 			onBlur={ onBlur }
 			onChange={ onChange }
 			onFocus={ onFocus }
+			onKeyDown={ onKeyDown }
 			ref={ ref }
-			step={ jumpStep }
```

Comment:
Just noting that the `step` prop is now being passed directly when spreading `{ ...props }`